test/dynamo/test_modes.py

```diff
@@ -93,7 +93,8 @@ def __torch_dispatch__(self, func, types, args, kwargs=None):
                 return func(*args, **kwargs)
 
         # test e2e, with Inductor, as smoketest.
-        @torch.compile(fullgraph=True, backend="inductor")
```

Comment:
Why this test? I'm curious haha

Reply:
It was a test that was added after `set_fullgraph` was merged - in that change, `fullgraph=True` went through a convert_frame.py path that increments `counters["frames"]["total"]`. This PR reverted that change so that the counter is no longer incremented.